### ISSUE TITLE
Process for updating linked.data.gov.au/def/ resources

### ISSUE THREAD

**Issue by dr-shorthair** [AGLDWG/governance#1]
We need a formal process for managing releases of AGLDWG resources, particularly the /def/ resources (vocabularies, ontologies) which are managed on behalf of the AGLDWG community as a whole, and not just datasets, each of which is clearly managed by an agency.

**Comment by edmondchuc** [comment]
Hi @dr-shorthair are you asking for some kind of automated set up where when you edit certain ontologies and vocabularies in their respective repositories, they then get updated automatically on linked.data.gov.au?

**Comment by dr-shorthair** [comment]
Yes. There should be a review step before any merge into the head branch. But build should be automatic.
